src/Enums/RequestTypes.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Packback\Lti1p3\Enums;
+
+/**
+ * Supported request types which map to an error log message.
+ */
+enum RequestTypes
```

Comment:
This was strongly inspired by [this blog post](https://stitcher.io/blog/php-enums#enum-methods).